### PR TITLE
Add multiple parents callback support

### PR DESCRIPTION
**Pull request by skryukov**
Adds support for multiple parents callbacks

### CODE REVIEW

**Comment by ianwhite** [comment]
Hi Nikita,

Thanks! What’s the etiquette regarding force-pushing feature branches?

Cheers,
Ian

**Comment by flash-gordon** [comment]
feel free to force push at any time. Normally, it's your branch exclusively

**Comment by solnic** [comment]
@skryukov my understanding is that this should fix #209 right?

**Comment by skryukov** [comment]
@solnic actually no,  this is an addition to #191. I'll look into #209 soon =)

File: spec/integration/params/multiple_parents_schema_spec.rb [hunk 1]
```diff
@@ -10,17 +10,29 @@
...
     end
   end
 
   it 'inherits rules' do
     expect(form.(name: '', age: '').errors.to_h)
       .to eql(email: ['is missing'], age: ['must be filled'], name: ['must be filled'])
   end
+
+  it 'inherits callbacks' do
+    expect(form.(name: '').to_h).to eql(name: '', foo: 'foo', bar: 'bar')
```
Comment (solnic) [inline spec/integration/params/multiple_parents_schema_spec.rb]: Codacy Issue found: [Prefer the use of `lambda.call(...)` over `lambda.(...)`.](https://app.codacy.com/gh/dry-rb/dry-schema/pullRequest?prid=4764392)

**Comment by solnic** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/dry-rb/dry-schema/pullRequest?prid=4764392&bid=15883550)

**Comment by solnic** [comment]
closing in favor of #246 where I rebased it on top of latest master and it should work with new CI setup now

**Pull request closed without merging**